### DISCUSSION TITLE
default Login and pass

### DISCUSSION THREAD

**Discussion by maxim221** [johang/sd-card-images#24] [Q&A] [answered]
Hello! I wonder what is the default login and password. I googled hard and have tried everything I found, but with no success. Please help-)

**Comment by maxim221** [comment, marked as answer]
it is mentioned in img file! thank you

**Reply by threepwoodd** [reply]
Sorry I can't figure it out. Can someone please give me a hint?

**Reply by maxim221** [reply]
> Sorry I can't figure it out. Can someone please give me a hint? 

The password is a part of file name
